README.md

```diff
@@ -26,11 +27,19 @@ Service to backup and/or restore mysql databases to/from S3
 
 >**It's recommended that your S3 bucket have versioning turned on.**
 
+`B2_APPLICATION_KEY_ID` (optional) Backblaze application key ID
+
+`B2_APPLICATION_KEY` (optional) Backblaze application key secret
+
+`B2_BUCKET` (optional) Name of the Backblaze B2 bucket, e.g., _database-backups_. When `B2_BUCKET` is defined, the backup file is copied to the B2 bucket.
```

Comment:
I would put this before the other two and make it clearer that when this is defined, the following two variables must be defined.